components/SwipeRow.js

```diff
@@ -412,6 +412,8 @@ class SwipeRow extends Component {
             toValue,
             friction: this.props.friction,
             tension: this.props.tension,
+            restSpeedThreshold: this.props.restSpeedThreshold,
+            restDisplacementThreshold: this.props.restDisplacementThreshold,
```

Comment:
Do you need to add these to SwipeRow.defaultProps as well?
If they are `undefined` will the animation use the default value or is this problematic?

Reply:
Yes, I tested without adding these props to SwipeRow and it still works like before this PR.